### ISSUE TITLE
installation not working

### ISSUE THREAD

**Issue by werdahias** [donadigo/eddy#128]
I built and installed it from source. But installing a package does not work. Log:

```
(com.github.donadigo.eddy:249789): dconf-DEBUG: 01:39:58.982: watch_established: "/org/gtk/settings/file-chooser/" (establishing: 1)
(com.github.donadigo.eddy:249789): dconf-DEBUG: 01:39:59.075: change_fast
(com.github.donadigo.eddy:249789): dconf-DEBUG: 01:39:59.076: change_notify: /org/gtk/settings/file-chooser/window-position
(com.github.donadigo.eddy:249789): dconf-DEBUG: 01:40:02.389: change_fast
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:02.398: role now get-details-local
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:02.588: emit transaction-list-changed
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:03.223: adding state 0x555ab441d610
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:03.223: role now resolve
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:03.223: emit transaction-list-changed
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:03.248: emit transaction-list-changed
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:03.744: remove state 0x555ab441d610
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:03.744: emit transaction-list-changed
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:05.870: adding state 0x7fad4401d9b0
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:05.870: role now install-files
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:05.901: using frontend gnome
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:05.912: remove state 0x7fad4401d9b0
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:05.912: emit transaction-list-changed
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:05.912: adding state 0x555ab441d550
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:05.912: role now resolve
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:05.939: emit transaction-list-changed
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:06.470: remove state 0x555ab441d550
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:06.470: emit transaction-list-changed
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:09.049: adding state 0x555ab4786180
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:09.049: role now install-files
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:09.085: using frontend gnome
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:09.095: remove state 0x555ab4786180
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:09.095: emit transaction-list-changed
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:09.095: adding state 0x555ab474bbe0
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:09.095: role now resolve
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:09.126: emit transaction-list-changed
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:09.657: remove state 0x555ab474bbe0
(com.github.donadigo.eddy:249789): PackageKit-DEBUG: 01:40:09.658: emit transaction-list-changed
(com.github.donadigo.eddy:249789): dconf-DEBUG: 01:40:23.419: sync
```

**Comment by mmstick** [comment]
This is probably because it's not getting the PolicyKit dialog requesting for authentication

**Comment by BenMorant** [comment]
Hello !
Same here I guess (on Elementary OS 7, I installed the flatpak, version 1.3.2)

`(com.github.donadigo.eddy:2): PackageKit-DEBUG: 22:18:27.176: role now get-details-local
(com.github.donadigo.eddy:2): PackageKit-DEBUG: 22:18:27.204: emit transaction-list-changed
(com.github.donadigo.eddy:2): PackageKit-DEBUG: 22:18:27.961: emit transaction-list-changed
(com.github.donadigo.eddy:2): PackageKit-DEBUG: 22:18:27.962: adding state 0x55e98d357100
(com.github.donadigo.eddy:2): PackageKit-DEBUG: 22:18:27.962: role now resolve
(com.github.donadigo.eddy:2): PackageKit-DEBUG: 22:18:27.978: emit transaction-list-changed
(com.github.donadigo.eddy:2): PackageKit-DEBUG: 22:18:28.608: remove state 0x55e98d357100
(com.github.donadigo.eddy:2): PackageKit-DEBUG: 22:18:28.609: emit transaction-list-changed
(com.github.donadigo.eddy:2): PackageKit-DEBUG: 22:18:34.958: adding state 0x55e98d5de010
(com.github.donadigo.eddy:2): PackageKit-DEBUG: 22:18:34.959: role now install-files
(com.github.donadigo.eddy:2): PackageKit-DEBUG: 22:18:34.988: emit transaction-list-changed
(com.github.donadigo.eddy:2): PackageKit-DEBUG: 22:18:34.988: remove state 0x55e98d5de010
(com.github.donadigo.eddy:2): PackageKit-DEBUG: 22:18:34.989: adding state 0x55e98d536d70
(com.github.donadigo.eddy:2): PackageKit-DEBUG: 22:18:34.989: role now resolve
(com.github.donadigo.eddy:2): PackageKit-DEBUG: 22:18:35.003: emit transaction-list-changed
(com.github.donadigo.eddy:2): PackageKit-DEBUG: 22:18:35.650: remove state 0x55e98d536d70
(com.github.donadigo.eddy:2): PackageKit-DEBUG: 22:18:35.650: emit transaction-list-changed

`

**Comment by ryonakano** [comment]
Duplicate of #134 and should be fixed in 1.3.3: https://github.com/donadigo/eddy/issues/134#issuecomment-1666947908:

![スクリーンショット 2023-08-07 07 39 55](https://github.com/donadigo/eddy/assets/26003928/974f43e1-c3b6-4d70-8760-f822aba9a34b)

**Comment by donadigo** [comment]
The update fixing this is now available in AppCenter.
